src/test/java/jp/ikedam/jenkins/plugins/updatesitesmanager/DescribedUpdateSiteJenkinsTest.java

```diff
@@ -24,388 +24,122 @@
+import org.hamcrest.Matcher;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.jvnet.hudson.test.JenkinsRule;
 import org.jvnet.hudson.test.TestExtension;
 import org.jvnet.hudson.test.recipes.LocalData;
 import org.kohsuke.stapler.DataBoundConstructor;
 
-import com.gargoylesoftware.htmlunit.FailingHttpStatusCodeException;
-import com.gargoylesoftware.htmlunit.html.HtmlForm;
-import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.isA;
 
 /**
  * Tests for DescribedUpdateSite, concerned with Jenkins.
  */
-public class DescribedUpdateSiteJenkinsTest extends HudsonTestCase
-{
-    public static class DescribedUpdateSiteForConfigureTest extends DescribedUpdateSite
-    {
+public class DescribedUpdateSiteJenkinsTest {
+
+    @Rule
+    public JenkinsRule jRule = new JenkinsRule();
+
+    @Rule
+    public ExpectedException ex = ExpectedException.none();
+
+    public static class DescribedUpdateSiteForConfigureTest extends DescribedUpdateSite {
         private static final long serialVersionUID = -5159897288105253315L;
-        
-        private boolean editable = true;
-        
-        @Override
-        public boolean isEditable()
-        {
-            return editable;
-        }
-        
-        public void setEditable(boolean editable)
-        {
-            this.editable = editable;
-        }
-        
+
+
         private String testValue;
-        
-        public String getTestValue()
-        {
+
+        public String getTestValue() {
             return testValue;
         }
-        
+
         @DataBoundConstructor
-        public DescribedUpdateSiteForConfigureTest(String id, String url, String testValue)
-        {
+        public DescribedUpdateSiteForConfigureTest(String id, String url, String testValue) {
             super(id, url);
             this.testValue = testValue;
         }
-        
-        @TestExtension("testDoConfigure")
-        static public class DescriptorImpl extends Descriptor
-        {
+
+        @TestExtension("shouldShowAllDescribedSitesAsManaged")
+        static public class DescriptorImpl extends Descriptor {
             @Override
-            public String getDescription()
-            {
+            public String getDescription() {
                 return "DescribedUpdateSiteForConfigureTest";
             }
-            
+
             @Override
-            public String getDisplayName()
-            {
+            public String getDisplayName() {
                 return "DescribedUpdateSiteForConfigureTest";
             }
         }
     }
-    public void testDoConfigure() throws Exception
-    {
+
+    @Test
+    public void shouldShowAllDescribedSitesAsManaged() throws Exception {

```

Comment:
Hmm... I actually can't get the difference. I can't imagine a case only one of them fails. (both tests will fail if something wrong).
But I think it's OK if you like it.